test/validator/readme_test.dart

```diff
@@ -47,21 +45,32 @@ void main() {
   });
 
   group('should consider a package invalid if it', () {
-    test('has no README', () {
+    test('has no README', () async {
+      await d.validPackage.create();
+
       deleteEntry(p.join(d.sandbox, 'myapp/README.md'));
       expectValidationWarning(readme);
     });
 
     test('has only a .gitignored README', () async {
+      await d.validPackage.create();
       await d.git(appPath, [d.file('.gitignore', 'README.md')]).create();
       expectValidationWarning(readme);
     });
 
     test('has a primary README with invalid utf-8', () async {
+      await d.validPackage.create();
       await d.dir(appPath, [
         d.file('README', [192])
       ]).create();
       expectValidationWarning(readme);
     });
+
+    test('has only a non-primary readme', () async {
+      await d.validPackage.create();
+      deleteEntry(p.join(d.sandbox, 'myapp/README.md'));
+      await d.dir(appPath, [d.file('README.whatever')]).create();
```

Comment:
or test `README.mdown` and `README.markdown` those are alternative suffixes we're working to deprecate.

Reply:
Done